console/module/user/src/main/java/org/eclipse/kapua/app/console/module/user/server/GwtUserServiceImpl.java

```diff
@@ -326,24 +326,24 @@ public User call() throws Exception {
                     return userService.find(scopeId, user.getCreatedBy());
                 }
             });
-
-            User modifiedUser = KapuaSecurityUtils.doPrivileged(new Callable<User>() {
+            final User modifiedUser = KapuaSecurityUtils.doPrivileged(new Callable<User>() {
 
                 @Override
                 public User call() throws Exception {
                     return userService.find(scopeId, user.getModifiedBy());
                 }
             });
+
             if (user != null) {
                 gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userStatus", user.getStatus().toString()));
                 gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userName", user.getName()));
                 gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userDisplayName", user.getDisplayName()));
                 gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userEmail", user.getEmail()));
                 gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userPhoneNumber", user.getPhoneNumber()));
                 gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userCreatedOn", user.getCreatedOn()));
-                gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userCreatedBy", createdUser.getName()));
+                gwtUserDescription.add(new GwtGroupedNVPair("userInfo", "userCreatedBy", createdUser != null ? createdUser.getName() : "N/A"));
```

Comment:
Please use `null`, this is managed by https://github.com/eclipse/kapua/blob/develop/console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/shared/model/GwtGroupedNVPair.java#L35 automatically